cmd/entrypoint/namespaces_test.go

```diff
@@ -0,0 +1,41 @@
+// +build linux
+
+package main
+
+import (
+	"os/exec"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+// This isn't a great unit test, but it's the best I can think of.
```

Comment:
this is such a nitpick but i think if you start docstrings with anything other than the name of the function it's confusing for tools